config/yaml.go

```diff
@@ -314,14 +334,9 @@ func unmarshalYAMLValue(reader io.ReadCloser, value interface{}, lookUp lookUpFu
 // will be used
 //
 // TODO: what if someone wanted a literal ${FOO} in config? need a small escape hatch
```

Comment:
Move it to a different file :) It is hard to achieve this with expand, but sometimes you don't have an access to edit a file(e.g. if a config file is not controlled by you), so using a loader for this task seems preferable.